xDripG5/Transmitter.swift

```diff
@@ -8,14 +8,23 @@
 
 import Foundation
 import CoreBluetooth
+import HealthKit
 import os.log
 
 
 public protocol TransmitterDelegate: class {
+    func dequeuePendingCommand(for transmitter: Transmitter) -> Command?
+
+    func transmitter(_ transmitter: Transmitter, didFail command: Command, with error: Error)
+
+    func transmitter(_ transmitter: Transmitter, didComplete command: Command)
+
     func transmitter(_ transmitter: Transmitter, didError error: Error)
 
     func transmitter(_ transmitter: Transmitter, didRead glucose: Glucose)
 
+    func transmitter(_ transmitter: Transmitter, didRead calibration: Calibration)
```

Comment:
Instead of returning this value separately and expanding the surface area for clients to manage, it would be more natural for a `Calibration?` property to be added to `Glucose`.